New course, SDU:  KE824 - Biomolecular Simulations

JHavstein opened the pull request.

JHavstein committed:
Setting up

How to curl files from GitHub LFS?

JHavstein committed:
Minor improvements

JHavstein committed:
Fix title

JHavstein committed:
Minor adjustments

The simulation folder in the external repo now don't have whitespace in their names.

JHavstein committed:
Add outline

JHavstein committed:
Set flag for interactive testing

JHavstein committed:
Adjustments + remove initialization

JHavstein committed:
Update ke824-biomolecular-simulations__N540034102-app.yml

To do: Complete the description of `parameters:sim_var:options`.

JHavstein committed:
Add all simulations

JHavstein committed:
Minor formatting + title

JHavstein committed:
Implement recursive download of material

* Download folder with simulation materials incl. everything in sub-folders.
* Tested only at depth 1.

JHavstein committed:
Minor

JHavstein committed:
Fix PATH

JHavstein committed:
Fix invocation + misc.

JHavstein committed:
Formatting of output messages

JHavstein committed:
Update README.md

JHavstein committed:
Update README.md

JHavstein committed:
Update README.md

JHavstein committed:
Update README.md

JHavstein committed:
Update README.md

JHavstein committed:
Update README.md

JHavstein committed:
Update README.md

FedericaLoVerso commented on Courses/SDU/ke824-biomolecular-simulations__N540034102/README.md:
```diff
@@ -17,4 +17,4 @@ In addition to the usual GROMACS parameters (`Batch mode`, `Shell`, and `Enable
 
 > [!NOTE]
-> Using `Re-download simulation files` when a folder is mounted will irrecoverably overwrite the simulation files in the mounted folder. Files added by the user (i.e., files that are not present in the external GitHub repository) are not affected by the `Re-download simulation files`.
+> Using `Re-download simulation files` when a folder is mounted will irrecoverably overwrite the simulation files in the mounted folder. Files added by the user (i.e., files that are not present in the external GitHub repository) are not affected by the `Re-download simulation files` and will hence remain untouched.
```
Suggestion:

When using the option _Re-download simulation files_, *Simulation* folders included in a [mounted folder](../guide/submitting.md#select-a-folder) will be irrecoverably overwritten. Files added by the user (i.e., files that are not present in the external GitHub repository, and are not part of the selected *Simulation*) are not affected by the `Re-download simulation files` and will hence remain untouched.

FedericaLoVerso replied:
In addition:

line 5 is GROMACS


I would eliminate from line 9 to 13, you already refer to gromacs and we want to add only specific instruction for the students..

JHavstein committed:
Incorporate comments from code review

JHavstein committed:
Fix typo

JHavstein committed:
Incorporate comments from code review

JHavstein committed:
Fix linting

Shellcheck

JHavstein committed:
Add packages for inflation of diff archive file types

JHavstein committed:
Fix typo

JHavstein committed:
Merge branch 'main' into newcourse-sdu-biomolecular-simulations

JHavstein committed:
Update README.md

Himanshu/Andreas' revisions.

JHavstein committed:
Fix formatting

JHavstein committed:
Add `website`

Added link to the course in the `UCloud-Courses` github repo. The link will only work when this branch has been merged into `main`.

JHavstein commented:
I've pushed a fresh build of the Docker image to the Docker registry.

FedericaLoVerso reviewed (approved):
The course is fine.
I just wonder if one should mention in the doc about file preview feature on UCloud (they do not have to download the files e.g. pdf for the tutorial etc)

JHavstein committed:
Add extra simulation choice

JHavstein committed:
Fix some formatting
